### ISSUE TITLE
False positive on `unused_braces` lint?

### ISSUE THREAD

**Issue by vpzomtrrfrt** [Schniz/render.rs#18]
Using the macros in this module seems to trigger a lint failure:
```
warning: unnecessary braces around function argument
  --> render_tests/src/lib.rs:72:28
   |
72 |               <head><title>{title}</title></head>
   |                            ^^^^^^^ help: remove these braces
   |
   = note: `#[warn(unused_braces)]` on by default
```

I assume this is not supposed to happen, since removing the braces doesn't work

**Comment by Schniz** [comment]
I agree. That’s because of the generated code. I have a POC that silences some of the errors - but I find it weird that these stylistic linting rules even run on generated code. We need to dig into that and fix the warnings it generates 😳

**Comment by daaku** [comment]
This has been fixed in 02548be3!

**Comment by Schniz** [comment]
Yup! Thanks for the hard work @daaku!
